lib/WPA/Andersen.cpp

```diff
@@ -811,6 +826,25 @@ void Andersen::updateNodeRepAndSubs(NodeID nodeId, NodeID newRepId)
     consCG->resetSubs(nodeId);
 }
 
+void Andersen::cluster(void) const
+{
+    assert(Options::MaxFieldLimit == 0 && "Andersen::cluster: clustering for Andersen's is currently only supported in field-insesnsitive analysis");
+    Steensgaard *steens = Steensgaard::createSteensgaard(pag);
```

Comment:
Why we need a steensgaard here?

Reply:
We need data to perform clustering.

Reply:
I see. This is to use steensgaard to boost andersen?

Reply:
Exactly